docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.        ax6.set_xlabel("nfe")
</pre><p>To make the plot clearer, only set the <code>.set_xlabel("nfe")</code> on the last plot, since the x-axis is shared in this plot. So remove that line for ax1 to ax5, and only keep for ax6.</p><p> </p><p>Also add a legend to this plot, or explain that each line is a different seed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>I don't want to have to rerun everything, so prefer not to add the legend for now</p>